### ISSUE TITLE
RSS feed please

### ISSUE THREAD

**Issue by schneems** [paketo-buildpacks/blog#69]
Providing an RSS feed allows people who care about buildpacks to get content pushed to their feed without having to poll the site manually.

I'm not familiar with Hugo, but it has docs on providing an RSS feed https://github.com/gohugoio/hugo/blob/master/docs/content/en/templates/rss.md

**Comment by pacostas** [comment]
https://blog.paketo.io/index.xml

**Comment by dmikusa** [comment]
Good catch @pacostas. I've been using the blog feed for a while, it works great.
